qserver/qs-config.yml

```diff
@@ -0,0 +1,51 @@
+# Bluesky-Queueserver start-re-manager configuration
+# use:
+#   queue-monitor &
+#   ./qs/qs_host.sh start
+#
+#   or
+#   cd ./qs
+#   start-re-manager --config=./qs-config.yml
+#
+# https://blueskyproject.io/bluesky-queueserver/manager_config.html
+
+network:
+    redis_addr: localhost:6379
```

Comment:
Specific for XPCS:

```suggestion
    redis_addr: amber:6379
```

Reply:
Specific for XPCS, we'd change `*` here to the IP number of the only workstation (amber) where we allow queueserver to run.  Want to be certain we don't try to start the QS host process on other workstations.  We need to avoid two (or more) of these for the same instrument.

Reply:
Same for next line, as well.

This is likely a configurable parameter for the copier template.